api/filters/basic_authentication.go

```diff
@@ -85,6 +85,7 @@ func (a *BasicAuthenticator) Authenticate(request *http.Request) (*web.UserConte
 		},
 		AuthenticationType: web.Basic,
 		Name:               username,
+		AccessLevel:        web.GlobalAccess,
```

Comment:
well.. yes and no... For now this might be ok

Reply:
Should the authenticators set that access level at all or should it be left to the authorizers?

Reply:
I was really thinking about it (how it should look for OSS SM). Probably authenticators should put the default access level (zero value)